documentation/book/ref-loggers-KafkaMirrorMaker.adoc

```diff
@@ -3,7 +3,7 @@
 // assembly-deployment-configuration-kafka.adoc
 
 [id='ref-loggers-KafkaMirrorMaker-{context}']
-= Kafka Mirror Maker loggers
+.Kafka Mirror Maker loggers
```

Comment:
It is important to be able to link here.

```suggestion
= Kafka Mirror Maker loggers
```

Reply:
I looked at this and decided to switch the section around with the Logging concepts. I thought it made more sense to list the loggers and then show how you might configure them. We can reuse the Logging concepts for other components, so I left the property value as logger.name: "INFO" in the example. We use the actual logger in the procedure that introduces the chapter.

Reply:
The problem is that I have seen several users to copy this blindly and wondering why does the log level not changes when they set it to 'logger.name: DEBUG`. So I'm not sure this is sufficiently clear to the users.

Reply:
Thanks. I thought to remove the inline example and link back to the sample we provide at the start of the chapter. In the end, I've removed the two snippets, added examples directly and amalgamated the two logging sections.